eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -84,6 +88,27 @@ stages:
       withPreviousSDK: false             # 🚫
 
   - ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
+
+    # CI - Stage 1 x64 legs ------------------------------------
```

Comment:
Looking for feedback on these header style comments.  I am attempting to improve the readability in making it easier to quickly differentiate the various types of legs.